src/main/java/uk/gov/di/config/RelyingPartyConfig.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.di.config;
+
+public class RelyingPartyConfig {
+    public static final String CLIENT_ID="xpCRKVpmLFtDS3aOaiNAmsgnm2UNnEEh";
+    public static final String OP_AUTHORIZE_URL ="https://dev-g98b2qxy.eu.auth0.com/authorize";
```

Comment:
Are we happy to merge + deploy these real Auth0 urls, or can we set up the routes on `https://di-auth-oidc-provider.london.cloudapps.digital/` first and use those?

They could also be env-vars set up in the PaaS config.

Reply:
This PR won't be merged as is - its just so there is a base and i agree we should point to the proper ui's in paas config and make the app default to localhost for local dev work

Reply:
updated config